BinaryEncoding.md

```diff
@@ -65,20 +65,36 @@ Yes:
 
 ## Global structure
 
-* A module contains:
-  * a header followed by
-  * a table (sorted by offset) containing, for each section, its type and offset (within the module), followed by
-  * a sequence of sections.
+* A module contains (in this order):
+  * A header
```

Comment:
You caught me. I don't know what the headers contain. At the very least, the module header contains the magic number, but besides that I don't really have anything in particular decided. Maybe some things like whether heap is 32/64 bit, source language (for ABI), and entry point. This will need to be figured out, but based on the level of detail in the rest of our design docs I'm not sure how much detail to go in here.

Maybe I should just mention some things like this as "ideas" for what a header would contain?